src/assets/images/LoginImage.jsx

```diff
@@ -1,5 +1,6 @@
 import React from "react";
 
+// eslint-disable-next-line no-unused-vars
 function LoginImage(props) {
```

Comment:
These Props are unused so I have disabled linting on this temporarily. I'm unsure why it's put there?

Reply:
Can just remove the props